vmcnet/models/equivariance.py

```diff
@@ -232,6 +247,14 @@ def setup(self):
             self.ndense, kernel_init=self.kernel_initializer_2e, use_bias=False
         )
 
+        self._attention_1e = SelfAttention(
```

Comment:
Can we do an if/else here based on use_transformer to either create the _attention_1e layer or the mixed_dense layer, but not both?

Reply:
make sense to me